lib/bap_llvm/bap_llvm_ogre_elf.ml

```diff
@@ -0,0 +1,86 @@
+open Core_kernel.Std
+open Bap.Std
+open Monads.Std
+
+open Bap_llvm_ogre_types
+
+module Scheme = struct
+  open Ogre.Type
+
+  (** flags that describes an entry behavior *)
```

Comment:
there is no need to duplicate comments in the `ml`, that are already in the `mli`.